daemon/api_aspects.go

```diff
@@ -76,7 +76,8 @@ func getAspect(c *Command, r *http.Request, _ *auth.UserState) Response {
 
 	// no results were found, return 404
 	if len(results) == 0 {
-		return NotFound("no fields were found")
+		errMsg := fmt.Sprintf("cannot get fields %s of aspect %s/%s/%s", strutil.Quoted(fields), account, bundleName, aspect)
```

Comment:
Nitpick: Seems little strange print the list the complete fields. It has the same feel as the [internal error for missing field](https://github.com/snapcore/snapd/pull/12953/files#diff-552af584ab0034353f8f34499ad35458d47a63c082bfbb5ed551c54e0b3fd29dR77), but these does not get exposed ever.

Reply:
I'm not sure I understand, we do expose the missing field in a very similar way?

Reply:
I was just considering how the message will look if there are many fields. Its not new information, the user send us all the fields and now we report it back. But for the typical case it would look nice.